lib/jsdom/living/helpers/html-constructor.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable global-require */
```

Comment:
Why?

Reply:
Good catch, forgot to remove this in a previous refactor.